be/src/storage/lake/meta_file.cpp

```diff
@@ -637,6 +641,17 @@ Status get_del_vec(TabletManager* tablet_mgr, const TabletMetadata& metadata, co
         ASSIGN_OR_RETURN(rf, fs::new_random_access_file(opts, tablet_mgr->delvec_location(metadata.id(), delvec_name)));
     }
     RETURN_IF_ERROR(rf->read_at_fully(delvec_page.offset(), buf.data(), delvec_page.size()));
+    // check crc32c
+    uint32_t crc32c = crc32c::Value(buf.data(), delvec_page.size());
```

Comment:
consider move the crc32c calculation into the if condition. the value will be calculated and be checked only when delvec_page.has_crc32().

Reply:
done